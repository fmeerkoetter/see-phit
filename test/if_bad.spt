R"*(
<if con=0>
  <div>
    a
  </div>
</if>

)*"_html; 
