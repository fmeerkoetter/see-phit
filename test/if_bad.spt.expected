Error() [with int ROW = 2; int COL = 10; WHAT = spt::Invalid_syntax_in_if_tag]':
